string node_name
string node_executable
string node_package
string node_machine
